### PR TITLE
Carlos pull

### PR DESCRIPTION
**Pull request by kgbraw**
Please review this, @casey-collab

### CODE REVIEW

File: espanol/torre/espera/cerrada/cerrada.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (casey-collab) [inline espanol/torre/espera/cerrada/cerrada.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by kgbraw** [comment]
Look at the changes @casey-collab

**Comment by casey-collab** [comment]
Looks great! :star:

**Pull request merged**